src/components/Tabs/types/tab.tsx

```diff
@@ -0,0 +1,5 @@
+export interface Tab {
+  id: string,
```

Comment:
Consider using semicolons instead of commas to separate properties in an interface. 🧐